readthedocs/search/pagination.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework.pagination import PageNumberPagination
+
+
+class SearchPagination(PageNumberPagination):
+    page_size = 10
```

Comment:
We might want to default this higher, since we aren't paginating results on the search page. We probably want 25 here perhaps.